docs/appendices/release-notes/5.5.0.rst

```diff
@@ -0,0 +1,103 @@
+.. _version_5.5.0:
+
+==========================
+Version 5.5.0 - Unreleased
+==========================
+
+.. comment 1. Remove the " - Unreleased" from the header above and adjust the ==
+.. comment 2. Remove the NOTE below and replace with: "Released on 20XX-XX-XX."
+.. comment    (without a NOTE entry, simply starting from col 1 of the line)
+
+.. NOTE::
+
+    In development. 5.5.0 isn't released yet. These are the release notes for
+    the upcoming release.
+
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 4.0.2 or higher
+    before you upgrade to 5.5.0.
+
+    We recommend that you upgrade to the latest 5.4 release before moving to
+    5.5.0.
+
+    A rolling upgrade from 5.4.x to 5.5.0 is supported.
+    Before upgrading, you should `back up your data`_.
+
+.. WARNING::
+
+    Tables that were created before CrateDB 4.x will not function with 5.x
+    and must be recreated before moving to 5.x.x.
+
+    You can recreate tables using ``COPY TO`` and ``COPY FROM`` or by
+    `inserting the data into a new table`_.
+
+.. _back up your data: https://crate.io/docs/crate/reference/en/latest/admin/snapshots.html
+.. _inserting the data into a new table: https://crate.io/docs/crate/reference/en/latest/admin/system-information.html#tables-need-to-be-recreated
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Breaking Changes
+================
+
+None
+
+
+Deprecations
+============
+
+None
+
+
+Changes
+=======
+
+SQL Statements
+--------------
+
+None
+
+
+SQL Standard and PostgreSQL Compatibility
+-----------------------------------------
+
+None
+
+
+Scalar and Aggregation Functions
+--------------------------------
+
+None
+
+
+Performance and Resilience Improvements
+---------------------------------------
+
+None
+
+
+Data Types
+----------
+
+None
+
+
+Administration and Operations
+-----------------------------
+
+None
+
+
+Fixes
```

Comment:
I guess we can remove this part completely?
Fixes should go only to `5.4.1` (and in future, `5.4.2`, etc.)

Reply:
Yes 👍